src/component.js

```diff
@@ -108,4 +107,5 @@ var Resizeable = React.createClass({
 
 });
 
-module.exports = Resizeable;
+module.exports = Resizable;
+global.Resizable = Resizable;
```

Comment:
Thanks this looks perfect, except for this `global.Resizable`, what would be the purpose? Browserify would convert this into `window.Resizable` right? I'm highly reluctant to pollute the global scope, if a user wants to do it he can `window.Resizable = require('react-comonent-resizable')` himself...